_source/_posts/2020-09-15-reintroducing-joel-franusic.md

```diff
@@ -0,0 +1,334 @@
+you [edit the code of the Okta Sign-In Widget with instant feedback, from inside
+your web browser](https://developer.okta.com/live-widget/). However, that tool uses JavaScript, and since I'm focusing
+on Python and Go, I wanted to make an example of running Python from inside of
+the browser. So, I wrote a game for you to play and I have included this game below:
+
+
+# How to play
+
+The goal of this game is to write a Python function (`answer()`) that takes
+three numbers of a sequence as input (we label these `a`, `b`, and `c`) and
+returns the next number in that sequence.
+
+For example, let's imagine that the `answer(a, b, c)` function is given part of this
+integer sequence as input:
+
+<p style="height: 150px">
+  <a href="https://dilbert.com/strip/2001-10-25">
+    <img style="max-height: 90%; max-width:90%;" src="https://assets.amuniversal.com/321a39e06d6401301d80001dd8b71c47"/>
+  </a>      
+</p>
+
+In this case, `answer()` would be called with the first three numbers of the sequence, like so:
+`answer(9, 9, 9)`  — in this case the correct answer would be `9` and the code
+to "solve" this sequence might look something like this:
+
+    def answer(a, b, c):
+        return 9
+
+To win this game, your job will be to write a function that will be able to give
+the correct answer for **all** of the sequences that will show up on the
+checklist below. The checklist will update as you solve for each sequence.
+Update the Python code below to start playing!
+
+Sequence checklist:
+
+  <ul id="messages"></ul>
+  <div id="error" class="alert error"></div>
+  <div id="success" class="alert success">
+    You did it!
+    Reward yourself by <a href="https://oeis.org/wiki/Welcome">learning more about The On-Line Encyclopedia of Integer Sequences</a> ®
+    and why Donald Knuth says you can use it to
+    <a href="https://youtu.be/BxQw4CdxLr8?t=1187">"compute your way into the literature"</a>.
+  </div>
+  <textarea id="editor">def answer(a, b, c):
+    return True
+
+play(answer)</textarea>
+<script type="text/javascript">
+document.addEventListener("DOMContentLoaded", function(){
+    brython(1)
+});
+</script>
+
+Edit the code above to play. No need to press a "save" button or anything. Your code will run once you've
+stopped typing for a bit.
+
+
+# "That's interesting, I guess. Why should I care?"
+
+Perhaps you played the game, perhaps you didn't. But in either case, the reason
+for me including this game in this blog post might not be apparent. Well, I
+included the game above to show that not only is it fairly easy to run
+a non-JavaScript interpreter in the browser, it can be a lot of fun when you do.
+
+For this game, I wanted to make a something that:
+
+* Involves writing some mildly-complex code, something just a little bit more complicated than "hello world".
+* Runs a soon as you stop typing.
+* Teaches the player something new.
+* Shows that it's fairly easy, and fun, to run Python directly from a browser.
```

Comment:
I'm not sure why this doesn't show up as bullets in your post. Maybe switch to - instead of *?